vendor/verrpc/verrpc.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
```

Comment:
Personal preference nitpick: move addition to vendor into a separate commit? Nice to have these changes separated out a little.